src/agora/dozf.cc

```diff
@@ -103,6 +120,8 @@ void DoZF::ComputePrecoder(const arma::cx_fmat& mat_csi,
         arma::cx_fmat(cfg_->UeNum(), cfg_->NumChannels(), arma::fill::zeros));
   }
   mat_ul_zf = mat_ul_zf_tmp;
+  float rcond = arma::rcond(mat_csi.t() * mat_csi);
+  return rcond;
```

Comment:
The computation of the condition number seems to not cheap. It seems this number is only used in PhyStats. Can we only enable the computation when necessary?